howto/sockets.po

```diff
@@ -380,6 +423,12 @@ msgid ""
 "of a following message. You'll need to put that aside and hold onto it, "
 "until it's needed."
 msgstr ""
+"Une complication dont il faut être conscient : si votre protocole de "
+"conversation permet de renvoyer plusieurs messages dos à dos (sans aucune "
```

Comment:
Je crois que le sens est ici au contraire :
```suggestion
"conversation permet de renvoyer plusieurs messages les uns à la suite des autres (sans aucune "
```
https://dictionary.cambridge.org/fr/dictionnaire/anglais/back-to-back : _happening one after the other, with nothing between_

Reply:
bien mieux 👍